src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -66,6 +66,11 @@ let uniqueId = 0;
 })
```

Comment:
I initially thought this content does not currently appear in the docs, but I just realized that this content is for the single file attachment component while the file drop content is for the file attachment component. Right now, all of the content is appearing for both components. I'm not sure how to get some of it to appear for one and some of it to appear for the other. @Blackbaud-SteveBrush, do you know what we need to do to make only the appropriate content appear for each page?

Reply:
Also, not sure where to note it, but docs-tools is exposing `SkyFileAttachmentLabelComponent`. I don't see that in the existing docs. Should it be in both file attachment and single file attachment docs? And should we provide a description for it?

Reply:
My vote would be to create an issue and come back to it later.

Reply:
I created an issue for this at https://github.com/blackbaud/skyux-forms/issues/175.